modules/version.js

```diff
@@ -0,0 +1,20 @@
+var exec = require('child_process').exec;
+
+var hash;
+
+module.exports.init = function(b) {
+	exec('git rev-parse HEAD', function(error, stdout, stderr) {
```

Comment:
Please also add something like `require('../package/json').version` into the output.